styleguide/components/hero/animations.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CSSTransition from 'react-transition-group/CSSTransition';
+import astroStyles from './astronauts.css';
+import Stars from './stars';
+
+function getTransition(klass, appearKlass, appearActiveKlass) {
+  const baseClassName = klass;
+  const appearClassName = appearKlass;
+  const appearActiveClassName = appearActiveKlass;
+  return (
+    <CSSTransition
+      appear
+      key={baseClassName}
+      timeout={{
+        appear: 0,
+      }}
+      in={true}
+      classNames={{
+        appear: astroStyles[appearClassName],
+        appearActive: astroStyles[appearActiveClassName],
+      }}
+      unmountOnExit
+    >
+      <div className={astroStyles[baseClassName]} />
+    </CSSTransition>
+  );
+}
+
+export default function Animations(props) {
+  return (
+    <div>
+      <Stars slug={props.slug} />
+      {props.slug === 'overview' && [
```

Comment:
INFO: Since we're testing `props.slug` against three known values, I would normally use a `switch` statement for this:

```
{switch(props.slug) {
  case 'overview': [];
  case '...': [];
  case '...': [];
  default: ???;
}}
```

... assuming I have the syntax correct.